xdsl/dialects/memref.py

```diff
@@ -218,10 +218,10 @@ def verify_(self) -> None:
         if not isinstance(self.type, MemRefType):
             raise Exception("Global expects a MemRefType")
 
-        if self.initial_value and not isinstance(self.initial_value,
-                                                 DenseArrayBase):
-            raise Exception(
-                "Global expects an initial value with type DenseArrayBase")
+        #if self.initial_value and not isinstance(self.initial_value,
```

Comment:
Is this PR still WIP?

Reply:
There is only this issue left, which relates to the issue I mentioned above.